src/templates/partials/menu.html

```diff
@@ -0,0 +1,25 @@
+<div id="menu">
+  <div>
+    <a href="/"><img src="/assets/images/peta-sitcheff-logo.svg" alt="Peta Sitcheff logo" width="233" height="480"></a>
+    <a class="close-menu" href="#">Close</a>
+    <ul id="navigation-links">
+      <li><a href="/#story">About</a></li>
```

Comment:
No need for the leading `/` in the `/#story`. Suppose you are on `/contact` then you'd go to `/#story` instead of `/contact#story`